content/_getting-started/first-flow.md

```diff
@@ -1,172 +1,123 @@
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_0.png)
 
-To start click on *+Add new flow* button to load the integration flow designer.
+**OR** go to *Flows* and click "Add New Flow":
 
-The designer helps building the flows from the reusable integration components
-such as `triggers` and `actions`. Every integration flow starts with a `trigger`
-component to retrieve data from the source. To start let us find the
-*Petstore component* in the list of integration components.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_1.png)
 
-![Select our Petstore component](/assets/img/getting-started/first-flow/getting-started-flow-002.png "Select our Petstore component")
+This brings you to *Designer* page. Let's start with naming your Flow, which is optional, and adding the initial trigger:
 
-For that purpose type `petstore` in the search bar and you will find it. Next,
-click on *Choose Petstore API (Node.js)* button.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_4.png)
 
-![Create the credentials to access](/assets/img/getting-started/first-flow/getting-started-flow-003.png "Create the credentials to access")
+The platform prompts you to setup the first step. First you may give the step a special name. Then you can select the Component you want to use, by using search bar, Component list, or load new Component from Catalog:
 
-Next, you will see the page *Choose Credentials* where you need to click on
-*+connect new credentials* link to create your first credentials. Let us fill-in
-an input form for these two fields:
-*   **Account name** - you can leave this unchanged or provide your own.
-*   **API key** - please use `{{site.data.tenant.petStoreApiKey}}` to get access to our [Petstore API]({{site.data.tenant.petStoreDocs}}).
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_5.png)
 
-Click on *Save* to verify the credentials. The verification can take no more than
-30 seconds. Upon completion you will see the page to choose one of the 3 triggers
-available in the Petstore component.
+We want to use Simple Trigger, so we will start typing "simple" to the search bar, and find it faster. Then select it and click "Choose Simple trigger":
 
-![Select the trigger](/assets/img/getting-started/first-flow/getting-started-flow-004.png "Select the trigger")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_6.png)
 
-Typically a component can have more than one triggers to retrieve different
-objects from the source API. Here we use the Petstore component to learn the
-{{site.data.tenant.name}} platform. It provides 3 different triggers to retrieve
-pets by their status. These 3 triggers differ by the implemented code style. So
-select any of them, for example click on the *Get Pets By Status With Promises*
-radio button to continue.
+The next step is [data sample](/data-sample-overview) retrieval. Click "Retrieve sample from Simple trigger" and then "Continue" to proceed:
```

Comment:
Maybe we should mention here that we don't want to discuss samples here as it goes beyond simple introduction? Also a link to sample docs for more details would help

Reply:
Line 36 has a link.